### DISCUSSION TITLE
Is this addon still being maintained?

### DISCUSSION THREAD

**Discussion by JRDK92** [AdiAddons/AdiBags#946] [General]
I prefer this addon over bagnon or any other (arkinventory etc). But seems like it's not updated in a while.... It doesn't work sometimes since it's out of date. Will it be updated?

**Comment by SpareSimian** [comment]
The addon works great. I'm not having any problems with it. It doesn't _need_ maintenance. If you've got a specific problem, file an issue about it. "It doesn't work" doesn't help anyone fix anything.
